### PR TITLE
Update row-level-security-report-server.md

### PR DESCRIPTION
**Pull request by gboreki**
Updating the limitations section to clearly indicate Custom Auth and Basic Auth are not supported for role assigments through group memberships.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@gboreki : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/report-server/row-level-security-report-server.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ If you're using integrated security with DirectQuery, then your users may notice
 
 Report authors don't have access to view the report data in Power BI Report Server until they've assigned themselves roles accordingly after uploading the report. 
 
+Role assignments through group memberships are only supported when the Power BI Report Server is configured to run with NTLM or Kerberos authentication. Servers running with Custom authentication or Windows Basic need explicitly users assigned to roles.
```
Comment (maggiesMSFT) [inline powerbi-docs/report-server/row-level-security-report-server.md:93]:
```suggestion
Role assignments through group memberships are only supported when Power BI Report Server is configured to run with NTLM or Kerberos authentication. Servers running with Custom authentication or Windows Basic need users explicitly assigned to roles.
```

**Comment by PRMerger8** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by maggiesMSFT — APPROVED** [review]

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**